### ISSUE TITLE
[🐛] App crashes after take a picture or select an image from gallery

### ISSUE THREAD

**Issue by mobtex-caio** [react-native-image-picker/react-native-image-picker#2060]
### Description

App crashes after I take a picture or select an image from gallery.
I check the permissions before the request

```
const imageUri = await launchCamera({
        mediaType: 'photo',
        cameraType: frontCamera ? 'front' : 'back',
      })
```

```
const fileList = await launchImageLibrary({
        mediaType: 'photo',
        selectionLimit: multiple ? 0 : 1,
      })
```

### Additional Information

- Image Picker version: 4.10.2
- React Native version: 0.69.3
- Platform: Android
- Development Operating System: MacOS
- Dev tools: VS Code

**Comment by BianXuerui** [comment]
请问 解决了吗？

**Comment by mobtex-caio** [comment]
> 请问 解决了吗？

No. I changed to https://github.com/baronha/react-native-multiple-image-picker for now.

**Comment by valn1** [comment]
i am having a similar issue, my app crashes about 40% of the time when i take a picture using this lib, but only crashes in certain devices

**Comment by Goostavo** [comment]
Possibly duplicate of #1972

**Comment by nitin41** [comment]
Facing same issue not on all the devices but on few devices.

**Comment by luisresendiz0** [comment]
I have the same issue, steps to reproduce:
1. Open the camera
2. Take a photo
3. Press the OK button to save the image
Then, app crashes and not return some error message to handle It...

Version:
- "react-native-image-picker": "^7.0.0"
- "react-native": "0.71.8"

Devices: 
- Motorola Moto G51 5G (Android 11)
- Redmi Note 7 (Android 10)

**Comment by vankhoa01** [comment]
Hi,
I am having a similar issue, but only crashes on Customer devices.
I can not reproduce this issue on my devices.
Do you have any ideas for this ?
Thank you so much
